api/jsonrpc/client.go

```diff
@@ -235,3 +236,55 @@ func Wait(ctx context.Context, interval time.Duration, check func(ctx context.Co
 	}
 	return ctx.Err()
 }
+
+type SimulatedActionResponse struct {
+	Output []byte
+	Error  error
+}
+
+type SimulateTransactionResponse struct {
+	SimulatedActions []SimulatedActionResponse
+	ReadKeys         []string
+	WriteKeys        []string
+	AllocateKeys     []string
+	BlockHeight      uint64
+}
```

Comment:
Can we re-use the types specified in `server.go` as we do for the existing server/client implementations?